### PR TITLE
Installation test pipeline

### PR DESCRIPTION
**Pull request by chicm-ms**
Only test pip install for now, will add more installation test later.
This test is scheduled daily.

### CODE REVIEW

File: test/pipelines-it-installation.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  strategy:
+    matrix:
+      Python36:
+        PYTHON_VERSION: '3.6'
+
+  steps:
+  - script: |
+      python3 -V
+      python3 -m pip install --upgrade pip setuptools --user
+      python3 -m pip install --upgrade nni --user
```
Comment (liuzhe-lz) [inline test/pipelines-it-installation.yml]: This only tests released version right?

Comment (chicm-ms) [reply test/pipelines-it-installation.yml]: yes.

Comment (liuzhe-lz) [reply test/pipelines-it-installation.yml]: So when this test case fails, we have already published a broken release...

Comment (chicm-ms) [reply test/pipelines-it-installation.yml]: This is just a start point, we can add more later, for example:
1. pre-release testpypi test
2. make dev install
3. install for different platforms (python version, virtual envs)
4. anything related installation not covered by IT.

File: test/pipelines-it-installation.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    vmImage: 'windows-latest'
+  strategy:
+    matrix:
+      Python36:
+        PYTHON_VERSION: '3.6'
+
+  steps:
+  - script: |
+      python -V
+      python3 -m pip install --upgrade pip setuptools --user
```
Comment (SparkSnail) [inline test/pipelines-it-installation.yml]: why `python3`?

Comment (chicm-ms) [reply test/pipelines-it-installation.yml]: Thanks, fixed.

**Pull request merged**